src/Apps/Order2/Routes/Checkout/Components/DeliveryOptionsStep/Order2DeliveryOptionsStep.tsx

```diff
@@ -39,3 +98,15 @@ export const Order2DeliveryOptionsStep: React.FC<
     </Flex>
   )
 }
+
+const FRAGMENT = graphql`
+  fragment Order2DeliveryOptionsStep_order on Order {
+    internalID
+    selectedFulfillmentOption {
```

Comment:
If we are going for the data from the server - once shipping step is complete - we can also rely on `ShippingLine` we get for the summary?

Reply:
Makes sense! Maybe a metaphysics change...